Makefile

```diff
@@ -0,0 +1,9 @@
+NAME      := simul
+SRC_EXT   := gz
+TEST_PACKAGES := $(NAME) $(NAME)-mpich $(NAME)-openmpi3
+
+include packaging/Makefile_packaging.mk
+
+test:
+	$(call install_repos,$(NAME)@$(BRANCH_NAME):$(BUILD_NUMBER))
+	yum -y install $(NAME)
```

Comment:
This is already provided by https://github.com/daos-stack/simul/blob/0e45b076d23b192daa5630fddc72bfb84751486f/packaging/Makefile_packaging.mk#L378-L381 so should be removed from here.

```suggestion
```

Reply:
Removing duped code